data-model/data-mapping/0.4.0/sssom_abcdefg.tsv

```diff
@@ -0,0 +1,164 @@
+dwc:group	Group	rdf property	ods:GeologicalContext	http://www.w3.org/2004/02/skos/core#exactMatch	abcd-efg:earthScienceSpecimen/unitStratigraphicDetermination/lithostratigraphicAttributions/lithostratigraphicAttribution/0/group	rdf property	LithostratigraphicAttribution	Takes the first lithostratigraphicAttribution	https://orcid.org/0000-0002-5669-2769	https://dissco.eu	2025-10-10
+dwc:formation	Formation	rdf property	ods:GeologicalContext	http://www.w3.org/2004/02/skos/core#exactMatch	abcd-efg:earthScienceSpecimen/unitStratigraphicDetermination/lithostratigraphicAttributions/lithostratigraphicAttribution/0/formation	rdf property	LithostratigraphicAttribution	Takes the first lithostratigraphicAttribution	https://orcid.org/0000-0002-5669-2769	https://dissco.eu	2025-10-10
+dwc:member	Member	rdf property	ods:GeologicalContext	http://www.w3.org/2004/02/skos/core#exactMatch	abcd-efg:earthScienceSpecimen/unitStratigraphicDetermination/lithostratigraphicAttributions/lithostratigraphicAttribution/0/member	rdf property	LithostratigraphicAttribution	Takes the first lithostratigraphicAttribution	https://orcid.org/0000-0002-5669-2769	https://dissco.eu	2025-10-10
+dwc:bed	Bed	rdf property	ods:GeologicalContext	http://www.w3.org/2004/02/skos/core#exactMatch	abcd-efg:earthScienceSpecimen/unitStratigraphicDetermination/lithostratigraphicAttributions/lithostratigraphicAttribution/0/bed	rdf property	LithostratigraphicAttribution	Takes the first lithostratigraphicAttribution	https://orcid.org/0000-0002-5669-2769	https://dissco.eu	2025-10-10
+dwc:typeStatus	Type Status	rdf property	ods:Identification	http://www.w3.org/2004/02/skos/core#exactMatch	abcd:specimenUnit/nomenclaturalTypeDesignations/nomenclaturalTypeDesignation/*/typeStatus	rdf property	Identification	When abcd:specimenUnit/nomenclaturalTypeDesignations/nomenclaturalTypeDesignation/*/typifiedName/fullScientificNameString matches the scientific name in the first TaxonIdentification of this Identification	https://orcid.org/0000-0002-5669-2769	https://dissco.eu	2025-10-10
+dwc:dateIdentified	Date Identified	rdf property	ods:Identification	http://www.w3.org/2004/02/skos/core#exactMatch	date/dateText	rdf property	Identification		https://orcid.org/0000-0002-5669-2769	https://dissco.eu	2025-10-10
+ods:isVerifiedIdentification	Is Verified Identification	rdf property	ods:Identification	http://www.w3.org/2004/02/skos/core#exactMatch	preferredFlag	rdf property	Identification		https://orcid.org/0000-0002-5669-2769	https://dissco.eu	2025-10-10
+dwc:identificationRemarks	Identification Remarks	rdf property	ods:Identification	http://www.w3.org/2004/02/skos/core#exactMatch	notes/value	rdf property	Identification		https://orcid.org/0000-0002-5669-2769	https://dissco.eu	2025-10-10
```

Comment:
is there a reason you don't use the abcd(efg) prefix for the object ID for lines 84-129?